foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -55,6 +55,17 @@
             ]
           }
         ]
+      },
+      {
+        "name": "enable-habushu-build-cache-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_7_0.EnableHabushuBuildCacheMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "*-pipelines/**/pom.xml"
```

Comment:
S: I think we just want this to be `pom.xml`.  Certainly we don't want to limit it to just the pipelines directory. They may have custom Habushu modules, and they may have data record modules.

Reply:
Done